apps/editing-toolkit/editing-toolkit-plugin/error-reporting/index.js

```diff
@@ -24,7 +24,10 @@ function activateSentry() {
 
 	// We still need to report the head errors, if any.
 	headErrors.forEach( ( error ) => Sentry.captureException( error ) );
-	Sentry.flush().then( () => delete window._jsErr );
+	Sentry.flush().then( () => {
+		delete window._jsErr;
+		window.removeEventListener( window._headJsErrorHandler );
```

Comment:
Since the event listener also creates a new jsErr array when it doesn't exist, future onerror events could still be saved, causing the array to become very large

Reply:
Ah, the actual listener can re-initialize `window._jsErr`... good catch!

Reply:
Actually, my mistake! The script already deletes the event listener, just at the very end of the file.